NeuroML2CoreTypes/NeuroMLCoreCompTypes.xml

```diff
@@ -36,87 +36,194 @@
     <ComponentType name="rdf_RDF" description="Structured block in an _annotation_ based on RDF. See https://github.com/OpenSourceBrain/OSB_API/blob/master/python/examples/grancelllayer.xml">
         <Child name="rdf:Description" type="rdf_Description"/>
         <Text name="xmlns:rdf"/>
+        <Text name="xmlns:rdfs"/>
+        <Text name="xmlns:dcterms"/>
+        <Text name="xmlns:dc"/>
+        <Text name="xmlns:prism"/>
+        <Text name="xmlns:bqbiol"/>
+        <Text name="xmlns:bqmodel"/>
+        <Text name="xmlns:scoro"/>
+        <Text name="xmlns:collex"/>
         <Dynamics/>
     </ComponentType>
 
 
     <ComponentType name="rdf_Description" description="Structured block in an _annotation_ based on RDF.">
 
-        <Child name="bqbiol:encodes" type="bqbiol_encodes"/>
-        <Child name="bqbiol:hasPart" type="bqbiol_hasPart"/>
-        <Child name="bqbiol:hasProperty" type="bqbiol_hasProperty"/>
-        <Child name="bqbiol:hasVersion" type="bqbiol_hasVersion"/>
```

Comment:
Not sure why these were Child elements in the first place given that they are all optional. But using Children doesn't seem to work---tests fail when parsing common files with/without annotations.

Something off with the parsing of Child/Children somewhere :(